src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.DesignTime.targets

```diff
@@ -29,6 +29,17 @@ Copyright (c) .NET Foundation. All rights reserved.
     <ProjectCapability Include="DotNetCoreRazorConfiguration" Condition="'$(_TargetingNETCoreApp30OrLater)'=='true'"/>
   </ItemGroup>
 
+  <!-- 
+    For now we need to treat component files as if they have a single file generator. This will allow us
+    to trigger a workspace update for the declaration files when they change.
+  -->
+  <ItemGroup>
```

Comment:
Putting all of this in the design-time targets because it will go away eventually. For now this works the same way as the Blazor 0.7 tooling.